ledger/acctonline_test.go

```diff
@@ -1253,7 +1254,9 @@ func TestAcctOnlineVotersLongerHistory(t *testing.T) {
 		delete(config.Consensus, testProtocolVersion)
 	}()
 
-	ml := makeMockLedgerForTracker(t, true, 1, testProtocolVersion, genesisAccts)
+	log := logging.TestingLogWithFilter(t, logging.DBLockedFilter)
+	log.SetLevel(logging.Warn)
```

Comment:
Why did this one get a `logging.Warn` setting, but others didn't?

Reply:
some tests use in-memory DB for performance and add lots of blocks. these emits lots of "locked" messages